Packs/Slack/Integrations/Slack/Slack.py

```diff
@@ -1719,7 +1569,7 @@ def set_channel_topic():
         channel_id = channel.get('id')
 
     if not channel_id:
-        return_error('Channel not found - the Demisto app needs to be a member of the channel in order to look it up.')
+        return_error('Channel not found - the Slack app needs to be a member of the channel in order to look it up.')
```

Comment:
I don't think "Slack app" is correct here. It should be XSOAR app.

Reply:
I think we need to figure out how we want to call it since it will be a customer's custom slack app and might confuse some users. I'll work on the language here.